.github/release.yml

```diff
@@ -0,0 +1,22 @@
+changelog:
+  exclude:
+    labels:
+      - skip-changelog
+      - documentation
+      - ci
+    authors:
+      - dependabot
+      - pre-commit-ci
+  categories:
+    - title: Added
+      labels:
+        - enhancement
+    - title: Changed
+      labels:
+        - BREAKING
```

Comment:
I think `BREAKING` should be a category of its own, but maybe "Breaking changes" is a more suitable name compared to "Changed", and it should be the first category / at the very top. Higher than "Added".